gradle/libs.versions.toml

```diff
@@ -40,6 +40,7 @@ paperlib = "1.0.8"
 paster = "1.1.6"
 vault = "1.7.1"
 serverlib = "2.3.6"
+linbus = "0.1.0-SNAPSHOT"
```

Comment:
0.1.0 was released as a non-snapshot version (yesterday?) - might be worth directly switching (https://github.com/EngineHub/WorldEdit/commit/203236d22a98de1251d0f1583da4b9a7aea37dde)